examples/npm/discord/commands/eval.js

```
@@ -0,0 +1,37 @@
+    let buf = '';
+    const sink = { write(s) { buf += s; return true; } };
+    const captured = new console.Console({ stdout: sink, stderr: sink });
+
+    const realConsole = globalThis.console;
+    globalThis.console = new Proxy(captured, {
+      get: (target, prop) => prop in target ? target[prop] : realConsole[prop],
+    });
+
+    let body;
+    try {
+      const result = await eval(code);
```

Biome (2.4.14)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)